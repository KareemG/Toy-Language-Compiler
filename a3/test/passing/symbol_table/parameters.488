
{
    procedure foo(a : integer, b : integer)
    {
        return
    }

    function fee(a : integer, b : integer) : integer
    {
        return with a + b
    }

    foo(2,3)
    fee(2,3)
}