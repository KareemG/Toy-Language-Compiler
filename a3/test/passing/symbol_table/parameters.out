
{
foo
{
a
b
}
fee
{
a
b
}
}